apiserver/facades/client/modelmanager/modelinfo_test.go

```diff
@@ -844,6 +847,16 @@ func (st *mockState) Clouds() (map[names.CloudTag]cloud.Cloud, error) {
 	return st.clouds, st.NextErr()
 }
 
+func (st *mockState) SetModelAgentVersion(newVersion version.Number, stream *string, ignoreAgentVersions bool) error {
+	st.MethodCall(st, "SetModelAgentVersion", newVersion, stream, ignoreAgentVersions)
```

Comment:
if we need this method, should we be recording the agent version so we can check later what it was?
or if it's just to satisfy an interface and not used by tests because we use gomock instead then we should return NotImplementedError
same below